db/task4/ddl.sql

```diff
@@ -0,0 +1,16 @@
+create extension if not exists "uuid-ossp";
+
+create table product (
+    id uuid primary key default uuid_generate_v4(),
+    image text,
+    title text,
+    description text,
+    price integer
+);
+
+create table stock (
+    id uuid primary key default uuid_generate_v4(),
+    product_id uuid,
```

Comment:
Also, if you want to have 1-1 dependency, you can also just use product_id as a primary and foreign key at the same time 💯